assets/src/dashboard/templates/raw/diy.json

```diff
@@ -459,58 +459,7 @@
       ],
       "type": "page",
       "id": "d47afc22-71b7-4303-9917-96f9a31ad38a",
-      "backgroundOverlay": "none",
-      "animations": [
-        {
-          "id": "d5e53845-e936-49a7-972f-7dcbac6a29df",
-          "type": "blinkOn",
-          "duration": 1250,
-          "delay": 0,
-          "direction": "normal",
-          "easing": "",
-          "fill": "forwards",
-          "iterations": "",
-          "blinkCount": 5,
-          "targets": [
-            "a0f67513-d98a-43f4-bb0b-ad3383051452",
-            "04ec92dc-313f-428d-87d0-00bd975290b5",
-            "59955c3a-57a0-4c4a-857d-bd6234e1382c"
-          ]
-        },
```

Comment:
Just prop warnings because `iterations` was set to an empty string, but our new logic removes fields that don't have an actual value.  

I don't recall if it was triggered when `enableAnimation` was set to false 🤔 ...